meetings/2025-01-29.md

```diff
@@ -0,0 +1,87 @@
+# W3C Solid Community Group: Weekly
+
+* Date: 2025-01-29T14:00:00Z
+* Call: https://meet.jit.si/solid-cg
+* Chat: https://matrix.to/#/#solid_specification:gitter.im
+* Repository: https://github.com/solid/specification
+* Status: Agenda
+
+
+## Present
+* [elf Pavlik](https://elf-pavlik.hackers4peace.net)
+* Hadrian Zbarcea
+* [Pierre-Antoine Champin](https://champin.net/#pa)
+* [Rahul Gupta](https://cxres.pages.dev/profile#i)
+* Theo
+* Michal
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+
+* elf Pavlik
+
+## Topics
+
+### [New Work Item] ActivityPub/Fediverse interoperability
+
+https://github.com/solid/specification/issues/708
+
+* eP: It is just an announcement, next week I hope to bring it up for a vote.
+* PAC: It is importantn to have people from Social Web CG also involved. We need consistent story about Solid and Activity Pub. 
+* HZ: Question to PAC, isn't it more important to bring it to LWS WG?
+* PAC: The charter mentions Social Web CG as a contact that we will synchronize with. There is no active Social Web WG. WGs have active scope, they need to sync, what we are discussin isn't directly in the scope of the LWS WG.
+* HZ: What is the best path forward to bring it to LWS
+* PAC: This is less urgent for the WG, at this moment it focuses on use cases. I don't expect strong coupling with Social Web CG. This work item should be a joined work in my opinion. Hopefully it will progress in CGs and we will have better understanding how to progress.
+* RG: The way we were discussin it yesterday, it will be layer on top of Solid, not something in the Solid spec. Everything should be workable as a layer on top of solid. 
```

Comment:
```suggestion
* HZ: What is the best path forward to bring it to LWS?
* PAC: This is less urgent for the WG; at this moment, the WG is focused on use cases. I don't expect strong coupling with Social Web CG. This work item should be a joint work, in my opinion. Hopefully it will progress in CGs, and we will have better understanding how to progress.
* RG: The way we were discussing it yesterday, it will be a layer on top of Solid, not something in the Solid spec. Everything should be workable as a layer on top of solid. 
```